DESCRIPTION

```diff
@@ -17,7 +17,6 @@ Depends:
 Imports:
     DBI (>= 0.3.0),
     httr (>= 0.6),
-    RCurl,
```

Comment:
Could you add `openssl` to the imports instead then? Otherwise things could break if httr moves away from openssl.